src/components/about-us/Timeline.tsx

```diff
@@ -22,6 +23,7 @@ const colors = {
 
 const data = [
   {
+    id: "2018",
```

Comment:
Very good call giving these an id! The only issue with that is that an id is a unique identifier for a data object. so it's strange that only some of them have one. For your purpose obviously not all of them can have one but only the first one in a year.  I'd call it "anchor" to make clear that it's not an actual data id but a property to help the UI. And personally, I'd put a comment somewhere that explains that as well because, it might not be super obvious on first sight for someone who's never seen the code or even for us if we look at it in 6 months and wonder about it.